### PR TITLE
Enhancement/NCC corporate theme

### PR DESCRIPTION
**Pull request by OliverSmelik**
Style Changes to service management/reporting for Scout Suite (not including dark mode yet)

### CODE REVIEW

**Comment by j4v** [comment]
Thanks for this @OliverSmelik, a few comments.

Text is off on finding dashboard:

sc_2019-05-02_12h52m15s

Also I get that red is the corporate colour but it's a bit aggressive - any way we can at least make the text highlights softer (maybe bold instead of make red?)?

**Comment by OliverSmelik** [comment]
Just fixing it now, the build ended up deleting some of the code I wrote. Will re-send once sorted.

**Comment by OliverSmelik** [comment]
Sorry about that, the way I have to work with causes a bit of confusion sometimes but sorted now. The nav styling wasn't added to the light theme only the dark. Added to both now and pushed.

**Comment by j4v** [comment]
> the way I have to work causes a bit of confusion

I know, I know :smile: 

Sorry for being picky, a few more comments.

"Normal" font for the Scout Suite link is smaller in the Dashboard (although bold): 

sc_2019-05-02_18h11m35s

And larger in other pages:

sc_2019-05-02_18h11m31s

Not crazy about the red colour on links:

sc_2019-05-02_18h09m32s

Particularly hard in dark mode:

sc_2019-05-02_18h19m36s

Also the "Danger" highlight (the `No` in the below screenshot) is hard to distinguish from the rest of the text:

sc_2019-05-02_18h25m36s

File: ScoutSuite/output/data/inc-scoutsuite/css/scoutsuite.css [hunk 1]
```diff
@@ -350,4 +369,131 @@ input:checked + .slider:before {
...
+}
+
+/* ------------------ Oliver CSS -------------------- */
+
+#account_id {
+    font-weight: bold;
+}
+
+#account_id .fa-cloud {
+    color: orange;
```
Comment (j4v) [inline ScoutSuite/output/data/inc-scoutsuite/css/scoutsuite.css]: Better keep that white, orange would hint to AWS and look a bit weird for GCP/Azure.

**Comment by OliverSmelik** [comment]
@j4v No problem, they're good suggestions as I'm just trying to feel out the NCC group 'Style' so a lot of trial and error with the design. I've made those changes and also added a scroll to the filters in the main navigation as the dropdown goes off the page for me.

**Comment by j4v** [comment]
Good stuff @OliverSmelik!

Is this expected:

sc_2019-05-04_21h07m07s

Looks a bit off, especially in dark mode.

Other then that everything else is :100:

**Pull request merged**